test_runner/regress/test_download_extensions.py

```diff
@@ -84,10 +44,37 @@ def upload_files(env):
 @pytest.mark.parametrize("remote_storage_kind", available_s3_storages())
 @pytest.mark.skip(reason="https://github.com/neondatabase/neon/issues/4949")
 def test_remote_extensions(
-    neon_env_builder: NeonEnvBuilder,
     remote_storage_kind: RemoteStorageKind,
     pg_version: PgVersion,
+    pytestconfig: Config,
+    test_output_dir: Path,
+    port_distributor: PortDistributor,
+    mock_s3_server: MockS3Server,
+    neon_binpath: Path,
+    pg_distrib_dir: Path,
+    default_broker: NeonBroker,
+    run_id: uuid.UUID,
 ):
+    alt_dir = test_output_dir / "pg_install"
+    log.warn(
+        f"Copying {pg_distrib_dir} to {alt_dir}. \
+            It is ~1.7GB so you might want to delete it from your test_output"
+    )
+    shutil.copytree(pg_distrib_dir, alt_dir)
+
+    neon_env_builder = NeonEnvBuilder(
+        repo_dir=Path(os.path.join(test_output_dir, "repo")),
+        port_distributor=port_distributor,
+        mock_s3_server=mock_s3_server,
+        neon_binpath=neon_binpath,
+        pg_distrib_dir=alt_dir,
+        pg_version=pg_version,
+        broker=default_broker,
+        run_id=run_id,
+        preserve_database_files=pytestconfig.getoption("--preserve-database-files"),
+    )
+
+    log.info(alt_dir)
```

Comment:
```suggestion
```

We already mentioned it above in `log.warn`

Reply:
ok